core/include/moveit/task_constructor/solvers/pipeline_planner.h

```diff
@@ -55,7 +55,21 @@ MOVEIT_CLASS_FORWARD(PipelinePlanner)
 class PipelinePlanner : public PlannerInterface
 {
 public:
-	PipelinePlanner();
+	struct Specification
+	{
+		moveit::core::RobotModelConstPtr model;
+		std::string ns{ "move_group" };
+		std::string pipeline{ "ompl" };
+		std::string adapter_param{ "request_adapters" };
+	};
+
+	static planning_pipeline::PlanningPipelinePtr create(const moveit::core::RobotModelConstPtr& model) {
+		return create(Specification{ .model = model });
```

Comment:
Isn't this line using the C++20 feature as well? (https://github.com/ros-planning/moveit_task_constructor/pull/249#issuecomment-808472365)

Reply:
Yes and no. It's the same feature, but apparently the version of gcc in CI *does* implicitly support it as long as all initialized members are *in order*, even if some in the end are not specified. The other use skipped some members and triggered the "sorry unimplemented" error.

I rewrote it here as well.